radio/src/CMakeLists.txt

```diff
@@ -336,6 +337,10 @@ if(FRSKY_STICKS)
   add_definitions(-DFRSKY_STICKS)
 endif()
 
+if(FONT)
```

Comment:
I don't like too much the option, I would prefer something -DFONT=SQT5 (with automatic conversion from SQT5 to sqt5)